AlfrescoApp/Model/Managers/AnalyticsManager.m

```diff
@@ -49,58 +58,275 @@ - (instancetype)init
 
 - (void)startAnalytics
 {
-    if ([[PreferenceManager sharedManager] shouldSendDiagnostics])
+    // Flurry Analytics
+    if (FLURRY_API_KEY.length > 0)
     {
-        [self start];
+        NSLog(@"FLURRY_API_KEY: %@", FLURRY_API_KEY);
+        
+        if ([[PreferenceManager sharedManager] shouldSendDiagnostics])
+            [self startAnalyticsType:AnalyticsTypeFlurry];
```

Comment:
Code standards: Don't write `if ()` statements without enclosing the conditional section in `{}`'s